include/lapi/rt_sigaction.h

```diff
@@ -190,8 +198,9 @@ static int ltp_rt_sigaction(int signum, const struct sigaction *act,
 		kact.k_sa_handler = act->sa_handler;
 		memcpy(&kact.sa_mask, &act->sa_mask, sizeof(sigset_t));
 		kact.sa_flags = act->sa_flags;
+#ifdef HAVE_SA_RESTORER
 		kact.sa_restorer = NULL;
-
+#endif
```

Comment:
I'm not sure about this part, shouldn't we ifdef this initialization only on ```#ifdef __mips__``` since the kernel may check that field is ```NULL``` if it's not used.

Otherwise the patch looks fine.